ignite/src/jepsen/ignite.clj

```diff
@@ -72,17 +72,14 @@
   [version]
   (reify db/DB
     (setup! [_ test node]
+      ;if we have no JDK on nodes, when install it. Works only with Debian/Ubuntu
+      (when-not (re-find #"JAVA_HOME" (get (shell/sh "printenv") :out)) (c/su (debian/install-jdk8!)))
       (info node "Installing Apache Ignite" version)
-      (c/su
-        (debian/install-jdk8!)
-        (cu/ensure-user! user)
```

Comment:
You can parameterize these sorts of calls based on the operating system. See, for example, https://github.com/jepsen-io/jepsen/blob/master/yugabyte/src/yugabyte/auto.clj#L40-L48